apps/etterlatte-proxy/src/main/kotlin/config/SoapSecurityMaskSensitiveHelper.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.etterlatte.config
+
+import org.apache.cxf.ext.logging.MaskSensitiveHelper
+import org.apache.cxf.message.Message
+import java.util.regex.Pattern
+
+/**
+ * Handle tag with multiline content between opening and end.
+ */
+internal class SoapSecurityMaskSensitiveHelper : MaskSensitiveHelper() {
+    private val matchPatternXML = Pattern.compile("(<wsse:Security.*?>)([\\s\\S.]*?)(</wsse:Security>)")
```

Comment:
- `MaskSensitiveHelper` støtter ikke å redigere innhold mellom to tags som går over flere linjer. Ikke funnet noen måte å overstyre på noe mer elegant vis.
- feltet `sensitiveProtocolHeaderNames` treffer normale headere, men ikke det som ligger innen soap:Header.

Reply:
Legge dette til som kommentarer i koden for fremtidige utviklere på teamet?